selftests/functional/plugin/tmtresult.py

```diff
@@ -0,0 +1,53 @@
+import re
+from os import path
+
+import yaml
```

Comment:
This makes PyYAML a development requirement, so it deserves to be listed in `requirements-dev.txt`.  And it should probably match the version in `optional_plugins/varianter_yaml_to_mux/setup.py`.